aladdin/lib/k8s/helm.py

```diff
@@ -69,6 +72,10 @@ def pull_packages(self, project_name, publish_rules, git_ref, extract_dir):
             if obj.key.endswith(f".{git_ref}.tgz")
         ]
 
+        if not package_keys:
+            logger.error(f"No published charts found for: {project_name} at {git_ref}")
+            sys.exit(1)
```

Comment:
early check to catch the case where `publish` has not happened yet, otherwise you get a verbose helm error instead